cwltool/update.py

```diff
@@ -231,13 +304,14 @@ def v1_2_0dev5to1_2(
     "v1.2.0-dev3": v1_2_0dev3todev4,
     "v1.2.0-dev4": v1_2_0dev4todev5,
     "v1.2.0-dev5": v1_2_0dev5to1_2,
+    "v1.3.0-dev1": v1_3_0dev1to1_3,
 }  # type: Dict[str, Optional[Callable[[CommentedMap, Loader, str], Tuple[CommentedMap, str]]]]
 
 
 ALLUPDATES = UPDATES.copy()
 ALLUPDATES.update(DEVUPDATES)
 
-INTERNAL_VERSION = "v1.2"
+INTERNAL_VERSION = "v1.3"
```

Comment:
```suggestion
INTERNAL_VERSION = "v1.3.0-dev1"
```

See the https://github.com/common-workflow-language/cwltool/blob/f0a844af6f93883fcacb3572a6366124a50d0909/cwltool/update.py#LL222